zulipterminal/ui_tools/views.py

```diff
@@ -950,14 +950,29 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 
 class StreamInfoView(PopUpView):
     def __init__(self, controller: Any, color: str,
-                 desc: str, title: str) -> None:
+                 desc: str, stream_id: int, title: str) -> None:
```

Comment:
Possibly for a separate refactor later (or before this), we could consider just passing in the `stream_id`, since we're already calling into the `model` to activate the muting, so could call in to fetch whatever details are required. We don't actually use the color right now.